pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -107,10 +28,20 @@ def prepare() {
         git url: env.OSA_OPS_REPO, branch: "master"
         sh "git checkout ${env.OSA_OPS_BRANCH}"
       }
-      dir("openstack-ansible-ops/${env.MULTI_NODE_AIO_DIR}") {
+      dir("openstack-ansible-ops/multi-node-aio") {
+        sh """#!/bin/bash
+          # The multi-node-aio tool is quite modest when it comes to allocating
+          # RAM to VMs -- since we have RAM to spare we double that assigned to
+          # infra nodes.
+          # TODO(mattt): When https://review.openstack.org/#/c/507875/ merges
+          #              we can simply overwrite infra_vm_server_ram in group vars
+          sed -i 's/server_vm_ram: 8192/server_vm_ram: 16384/g' playbooks/host_vars/infra*.yml
```

Comment:
the PR to set this via a variable has merged. we can now drop this injection and just use an override.